ReadSomeComments.txt

```diff
@@ -0,0 +1,3 @@
+
+I (Carlos Molina Jimenez) forked mirage-deck on my Mac Book air to include UCN logo and thanks on the slides.
+
```

Comment:
why is this file here? please remove from the PR. `git log` maintains all the history we need.